services/order.py

```diff
@@ -0,0 +1,34 @@
+import datetime
+
+from django.db import transaction
+from django.db.models import QuerySet
+
+from db.models import Order, User, Ticket
+
+
+def create_order(
+        tickets: list[dict],
+        username: str,
+        date: str = None
+) -> None:
+    with transaction.atomic():
+        user = User.objects.get(username=username)
```

Comment:
The `get` method will raise a `DoesNotExist` exception if no user is found with the given username. Consider handling this exception or using `get_object_or_404` if this function is used in a view context.